ecmascript/transforms/module/src/util.rs

```diff
@@ -20,8 +21,8 @@ use swc_ecma_visit::{Fold, FoldWith, VisitWith};
 
 pub(super) trait ModulePass: Fold {
     fn config(&self) -> &Config;
-    fn scope(&self) -> &Scope;
-    fn scope_mut(&mut self) -> &mut Scope;
```

Comment:
As far as I can tell, this change is necessary because we cannot return a `&Scope` without also passing ownership of the `Ref` that it comes from.